test/test_utils.rb

```diff
@@ -1,6 +1,21 @@
 require 'helper'
 
 class TestUtils < JekyllUnitTest
+  context "The \`Utils.deep_merge_hashes\` method" do
```

Comment:
Let me add, I don't quite remember if Minitest has descriptions in descriptions.